components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/internal/constant/Constants.java

```diff
@@ -41,14 +40,42 @@ public class Constants {
     public static final String RESOURCE_TYPE = "resourceTypeName";
     public static final String RESOURCE_NAME = "resourceName";
 
+    public static final String SESSION_EVENT_HOOK_NAME = "SessionEventHook";
+    public static final String SESSION_EVENT_HOOK_ENABLED = "SessionEventHook.enable";
+
+    public static final String CREDENTIAL_EVENT_HOOK_NAME = "CredentialEventHook";
+    public static final String CREDENTIAL_EVENT_HOOK_ENABLED = "CredentialEventHook.enable";
+
+    public static final String SESSION_ID = "sessionId";
+
     /**
      * Constants for event config names (keys).
      * These names will be equal to the config attribute keys stored in the core config store.
      */
     public static class EventHandlerKey {
 
-        public static final String LOGIN_SUCCESS_EVENT = "logins.loginSuccess";
-        public static final String LOGIN_FAILED_EVENT = "logins.loginFailed";
+        public static class WSO2 {
+
+            public static final String LOGIN_SUCCESS_EVENT = "logins.loginSuccess";
+            public static final String LOGIN_FAILED_EVENT = "logins.loginFailed";
+
+            private WSO2() {
+
+            }
+        }
+
+        public static class CAEP {
+
+            public static final String SESSION_REVOKED_EVENT = "caep.sessionRevoked";
```

Comment:
```suggestion
            public static final String SESSION_REVOKED_EVENT = "caep.sessionRevoked";
```

isn't this should be `session.sessionRevoked` ?